js/components.tsx

```diff
@@ -283,11 +285,11 @@ export const ProgressBar = forwardRef<
                 position={PositionType.Absolute}
                 width="100%"
                 height="100%"
-                left={12}
+                left={props.barLeftMargin ?? 12}
             >
                 {props.children}
             </Container>
-            {value > 0.05 && (
+            {value > 0.001 && (
```

Comment:
This was here to prevent visual artefacts when the value is less than the rounding of the panels. It should probably be based on the rounding 🤔

Reply:
I needed to think a bit about what you meant, but yes. It should be based on the rounding... if the value is too small you should still see a little sphere. It's a bit tricky since the value is a percentage, so it needs to be calculated on the current width too.